frontend/src/components/experimentRuns/ExperimentRunsTabs.tsx

```diff
@@ -28,21 +28,22 @@ const tabStyle = {
 	padding: "0 24px 24px 24px",
 	overflow: "auto",
 	height: "100%",
-  }
-
-function ExperimentRunsTabs() {
+}
 
-	const FREEZEMAN_TAB_KEY = 'freezeman'
-	const EXTERNAL_TAB_KEY = 'external'
+export enum ExperimentRunsTabsName  {
+    FREEZEMAN = 'freezeman',
+	EXTERNAL = 'external',
+}
 
-	const [activeKey, setActiveKey] = useHashURL(FREEZEMAN_TAB_KEY)
+function ExperimentRunsTabs() {
+	const [activeKey, setActiveKey] = useHashURL(ExperimentRunsTabsName.FREEZEMAN)
 	const token = useAppSelector(selectAuthTokenAccess)
 	const experimentRunsTableState = useAppSelector(selectExperimentRunsTable)
 	const actions = useAppSelector(selectExperimentRunsTemplateActions)
 
 
 	function getPageHeaderExtra() {
-		if (activeKey === FREEZEMAN_TAB_KEY) {
+		if (activeKey === ExperimentRunsTabsName.FREEZEMAN) {
```

Comment:
I think this function should be in a `useCallback`.

Reply:
this was initially part of the changes but after bashing my head on this task for several days, i decided to simply leave the enum instead of the initial freezeman key.